packages/graphiql/babel.config.js

```diff
@@ -1,13 +1,36 @@
-module.exports = {
+const babelEnvConfig = {
+  // ignore browserslist ONLY when we make our main/modules babel builds
+  ignoreBrowserslistConfig: Boolean(process.env.ESM && !process.env.BUNDLE),
+  modules: process.env.ESM ? false : 'commonjs',
+  targets: process.env.ESM ? 'last 1 chrome version, last 1 firefox version': undefined,
```

Comment:
Prettier should be detecting and handling this:

```suggestion
  targets: process.env.ESM ? 'last 1 chrome version, last 1 firefox version' : undefined,
```